include/lk/absyn.h

```diff
@@ -63,48 +65,59 @@ typedef std::equal_to<std::string> lk_string_equal;
 
 #define lk_tr(s) lk::get_translation(s)
 
-namespace lk
-{
-	lk_string get_translation(const lk_string &);
-	void set_translation_function(lk_string(*f)(const lk_string&));
+namespace lk {
+    lk_string get_translation(const lk_string &);
+
```

Comment:
How did these extra new lines get added?

Reply:
What do you mean-- as in why is it double spaced?

Reply:
Yes.